core/server/api_container/server/startosis_engine/plan_yaml/plan_yaml.go

```diff
@@ -8,11 +8,13 @@ const (
 	exec   TaskType = "exec"
 )
 
-type privatePlanYaml struct {
-	PackageId      string           `yaml:"packageId,omitempty"`
-	Services       []*Service       `yaml:"services,omitempty"`
-	FilesArtifacts []*FilesArtifact `yaml:"filesArtifacts,omitempty"`
-	Tasks          []*Task          `yaml:"tasks,omitempty"`
+type PlanYaml struct {
```

Comment:
Potentially worth a quick comment on why this struct is a keyed dict; as a new engineer I would have expected it to be a list of instructions. Obv I Kevin know the context (b/c we hacked this together for Ben), but might be worth jotting a quick note here